contracts/contracts/MerkleDrop.sol

```diff
@@ -121,8 +121,11 @@ contract MerkleDrop {
         droppedToken.burn(value);
     }
 
-    function decay(uint value, uint timeToDecay, uint totalDecayTime) internal pure returns (uint) {
+    function decay(uint value, uint timeToDecay, uint totalDecayTime, bool roundUp) internal pure returns (uint) {
         uint decay = value*timeToDecay/totalDecayTime;
+        if (decay * totalDecayTime != value * timeToDecay && roundUp) {
+            decay = decay + 1;
+        }
         return decay >= value ? value : decay;
```

Comment:
Please change the computation of `decay` to the following:
```
uint decay;
if (roundUp) {
    decay = (value * timeToDecay + totalDecayTime -1 ) / totalDecayTime
} else {
   decay = value*timeToDecay/totalDecayTime;
}
```
The way you round is quite unfamiliar to me and uses more operations. The changed code also only assigns `decay` once.